api/handlers_e2e_test.go

```diff
@@ -236,7 +236,7 @@ func TestAccountExcludeParameters(t *testing.T) {
 
 	const expectedAppIdx = 1 // must be 1 since this is the first txn
 	const expectedAssetIdx = 2
-	createAppTxn := test.MakeCreateAppTxn(test.AccountA)
+	createAppTxn := test.MakeCreateSimpleAppTxn(test.AccountA)
```

Comment:
I can understand push back for renaming this. Could have done just as well to keep the old name.